hypothesis-python/src/hypothesis/internal/compat.py

```diff
@@ -13,270 +13,53 @@
 #
 # END HEADER
 
-import array
 import codecs
 import importlib
 import inspect
-import math
 import platform
-import re
 import sys
 import time
-from base64 import b64encode
-from collections import namedtuple
+from typing import Tuple, Type  # noqa
```

Comment:
`Type` is an unused import and breaks compatibility with Python 3.5.0 and 3.5.1 - see #2314.